components/file_attachment.jsx

```diff
@@ -40,7 +47,7 @@ export default class FileAttachment extends React.PureComponent {
         super(props);
 
         this.state = {
-            loaded: Utils.getFileType(props.fileInfo.extension) !== 'image'
+            loaded: getFileType(props.fileInfo.extension) !== 'image'
```

Comment:
Not part of this PR, I know, but it drives me crazy that `getFileType` doesn't return constant-defined strings that could be compared against. Would be great if someone refactored that...

Reply:
Yeah, that makes sense, I'll submit separate PR to refactor that.

Reply:
PR submitted https://github.com/mattermost/mattermost-webapp/pull/701